roles/ceph-osd/tasks/system_tuning.yml

```diff
@@ -32,6 +32,32 @@
     when: ursula_os == 'rhel'
   when: ceph.disable_transparent_hugepage
 
+- name: create ursula tuned profile dir
+  file: path=/etc/tuned/ursula state=directory
+  when: ursula_os == 'rhel'
+
+# tune disk read-ahead for ceph
+# if there are more tunings in future,
+# we should move the common part(main section) into common role
+- name: set disk read-ahead for ceph
+  ini_file:
+    dest: /etc/tuned/ursula/tuned.conf
```

Comment:
can we call this file `/etc/tuned/ursula/ceph.conf` ?   other roles may eventually need to do similar.